src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -138,7 +142,7 @@ class ReimbursementAccountPage extends React.Component {
         // We grab the currentStep from the achData to determine which view to display. The SetupWithdrawalAccount flow
         // allows us to continue the flow from various points depending on where the user left off. We can also
         // specify a specific step to navigate to by using route params.
-        const currentStep = this.getStepToOpenFromRouteParams() || this.props.reimbursementAccount.achData.currentStep;
+        const currentStep = this.props.reimbursementAccount.achData.currentStep;
```

Comment:
I'm undoing this for now since I ran into an issue where we couldn't move past the `BankAccountStep` if we are on `/bank-account/new`. Not entirely sure how to get around this yet. Basically, we can have routes and let users navigate to specific steps based on them or we can infer what step the user should be on and then send them there. But I'm not sure if it makes sense to do both or how to work out the logic for it yet.